python/cuml/test/dask/test_random_forest.py

```diff
@@ -334,14 +335,66 @@ def test_rf_regression_dask_cpu(partitions_per_worker, cluster):
         X_train_df, y_train_df = dask_utils.persist_across_workers(
             c, [X_train_df, y_train_df], workers=workers)
 
-        cu_rf_mg = cuRFR_mg(**cu_rf_params)
-        cu_rf_mg.fit(X_train_df, y_train_df)
+        cuml_mod = cuRFR_mg(**cu_rf_params)
+        cuml_mod.fit(X_train_df, y_train_df)
 
-        cu_rf_mg_predict = cu_rf_mg.predict(X_test, predict_model='CPU')
+        cuml_mod_predict = cuml_mod.predict(X_test, predict_model='CPU')
 
-        acc_score = r2_score(cu_rf_mg_predict, y_test)
+        acc_score = r2_score(cuml_mod_predict, y_test)
 
         assert acc_score >= 0.67
 
     finally:
         c.close()
+
+
+@pytest.mark.parametrize('partitions_per_worker', [1])
```

Comment:
Is 1 necessary? Would be nice to test multiple partitions per worker.

Reply:
The 1's were added to create a dask rf test which can be used for apples to apples comparison with sklearn's predict proba

Reply:
I think for a unit test that will live on, it'd be good to have multiple partitions